guides/releasing/Release Workflow.md

```diff
@@ -0,0 +1,328 @@
+    
+**How:**
+
+- Go to `react-native-lab/react-native` submodule.
+- Coordinate with James (@ide) to create a new branch for new SDK in our `react-native` fork (`sdk-XX` typically, where `XX` is the major number of the SDK version).
+- Run `et update-react-native`. This expotools command copies `ReactAndroid` and `ReactCommon` folders from the submodule to the respective paths: `android/ReactAndroid` and `android/ReactCommon` and then executes `ReactAndroidCodeTransformer` that applies some Expo-specific code transformations.
+- Add your git changes from both `react-native-lab` and `android` folders and create a pull request to `master` branch.
+
+## 1.3. Versioning code for the new SDK
+
+| Requirements                                         |
+| ---------------------------------------------------- |
+| [1.2. Update React Native](#1-2-update-react-native) |
+
+**Why:** As we need to support multiple SDK versions, the code needs to be prefixed for each version so we don't get conflicts and duplicated classes. Such prefixed version of our APIs is called ABI.
+
+**How:**
+
+- **iOS**:
+  - Run `et add-sdk --platform ios` to copy unversioned code into the new ABI and prefix (or suffix) its files and corresponding code references with `ABIXX_0_0`. If this script errors partway, you can always delete the new directory it created under `ios/versioned-react-native` and revert any other changes it made to `EXSDKVersions.plist` and `sdkVersions.json` (or just run `et remove-sdk --platform --sdkVersion XX.0.0`). Then it's safe to run this script again.
+  - Let the `add-sdk` script to regenerate Podfile and reinstall pods, then try to build the project in Xcode. This script does most of the work, but usually breaks in various ways, partly because some assumptions change every SDK cycle. If you found anything broken, please keep versioning script up to date.
+  - Submit a pull request with versioned code to `master` branch.
+- **Android**:
+  - *todo*
+
+# Stage 2 - QA
+
+## 2.1. Quality Assurance
+
+| Requirements                                                                 |
+| ---------------------------------------------------------------------------- |
+| [1.3. Versioning code for the new SDK](#1-3-versioning-code-for-the-new-sdk) |
+
+**Why:** We really care about the quality of the code that we release for the users. Quality Assurance is the most important task during the release process, so please don't ignore any steps and also focus on things that have been changed/reworked/refactored in this cycle. 
+
+**How:**
+
+- Go through another guide about [Quality Assurance](Quality Assurance.md).
+
+## 2.2. Publish native-component-list app
+
+| Requirements                                     |
+| ------------------------------------------------ |
+| [2.1. Quality Assurance](#2-1-quality-assurance) |
+
+TODO
```

Comment:
should be published to both the `community` and `applereview` accounts with the new SDK version (not UNVERSIONED)

Reply:
Yes, but imho this should be done by a separate tool, so we can publish them both at once and don't have to remember the passwords. Also, people at swmansion still don't have an access to 1Password account so we cannot login into `community` account. 

My suggestion: put encrypted password of `community` and `applereview` to repository secrets and make a simple expotools command `et publish-ncl` that make sure the `sdkVersion` is not `UNVERSIONED` and then publish it to both accounts.